requirements-dev.txt

```
@@ -1,3 +1,4 @@
+asyncmock; python_version < '3.8'
```

```suggestion
mock; python_version < '3.8'
```

I think `mock` is what you want here. Do tests still pass?